.github/workflows/ci.yml

```diff
@@ -20,7 +20,8 @@ jobs:
         id: changes
         run: |
           echo "all=$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | xargs)" >> $GITHUB_OUTPUT
-          echo "c=$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | grep '\.\(c\|h\)$' | xargs)" >> $GITHUB_OUTPUT
+          # exclude src/scanner.c from the changed C code, because we'd normally review it.
+          echo "c=$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | grep '\.\(c\|h\)$' | grep -v 'src/scanner.c' | xargs)" >> $GITHUB_OUTPUT
```

Comment:
If we stop tracking changes at `src/scanner.c`, `needs.changedfiles.outputs.c` won't be set and tests in non-Linux environments won't be triggered. Is that a desired behavior?

Reply:
Gaa you're right. I guess I can make another category like `gen`

Reply:
Addressed in https://github.com/tree-sitter/tree-sitter-scala/pull/288/commits/5029432a9ec40ebc88b8aee2ef097678c437df06